src/nvim/getchar.c

```diff
@@ -1290,6 +1290,7 @@ openscript (
     int save_insertmode = p_im;
     int save_finish_op = finish_op;
     int save_msg_scroll = msg_scroll;
+    const int save_reg_executing = reg_executing;
```

Comment:
The patch modifies the function `save_current_state()`, included in vim-patch:8.0.1139 (toolbar).

Reply:
I'm wrong. vim-patch:8.1.0992 is for `save_current_state` so it shouldn't be merged.